kernel/Dockerfile

```diff
@@ -105,7 +119,7 @@ RUN make -j "$(getconf _NPROCESSORS_ONLN)" KCFLAGS="-fno-pie" && \
         ;; \
     esac && \
     cp System.map /out && \
-    ([ "${EXTRA}" = "-dbg" ] && cp vmlinux /out || true)
+    ([ "${DEBUG}" = "-dbg" ] && cp vmlinux /out || true)
```

Comment:
This should probably just check for it being set

Reply:
Like this, ([ -n "${DEBUG}" ] && cp vmlinux /out || true) ?